addon/templates/components/ember-sparkles/axis.hbs

```diff
@@ -0,0 +1,71 @@
+{{#with (hash
+  position=position
+  ticks=ticks
```

Comment:
IOW, we shouldn't assign properties to the hash if they're available in scope and we're not changing them.